general/administration/installing.md

```diff
@@ -37,142 +33,89 @@ The Jellyfin project and its contributors offer a number of pre-built binary pac
-```
-
-**Windows/MacOS**
-
-For windows and MacOS you can use the following link to install Docker for Desktop:
-
-<https://www.docker.com/products/docker-desktop>
-
-Now test the installation by running the following command in your terminal(MacOS) or Powershell(Windows):
+> [!Note]
+> For ARM hardware and RPi, it is recommended to use the LinuxServer.io image since hardware acceleration support is not yet available on the native image.
 
-```sh
-docker run hello-world
-```
+> [!Note]
+> Use host mode for networking in order to use DLNA or an HDHomeRun.
 
-You should see a message that states:
+### Docker
 
-```sh
-Hello from Docker! This message shows that your installation appears to be working correctly.
-```
+[Docker](https://www.docker.com/) allows you to run containers on Linux, Windows and MacOS.
 
-#### Preparing the Folders
+> [!Note]
+> There is currently an [issue](https://github.com/docker/for-linux/issues/788) with read-only mounts in Docker. If there are submounts within the main mount, the submounts are read-write capable.
 
-Before running the Jellyfin Container. We need to make 3 folders:
+The basic steps to create and run a Jellyfin container using Docker are as follows:
 
-- Config
-- Cache
-- Media
+1. Follow the [offical installation guide to install Docker](https://docs.docker.com/engine/install/).
 
-**Ubuntu/Mac**
+2. Download the latest container image:
 
-```sh
-mkdir -p Path/To/Config
-mkdir -p Path/To/Cache
-mkdir -p Path/To/Media
-```
+    ```sh
+    docker pull jellyfin/jellyfin
+    ```
 
-**Windows**
-Make a folder called Jellyfin, and create the 3 folders inside the Jellyfin Folder.
+3. Create persistent storage for configuration and cache data:
 
-These folders will be used by Jellyfin to store data in. By default everything inside a Docker container gets removed once you delete the container. By making these folders you are able to delete the Jellyfin container while keeping your data. Amazing!
+    Either create two persistent volumes:
 
-#### Running the Container
+    ```sh
+    docker volume create jellyfin-config
+    docker volume create jellyfin-cache
+    ```
 
-1. Get the latest image.
+    Or create two directories on the host and use bind mounts:
 
     ```sh
-    docker pull jellyfin/jellyfin
+    mkdir /path/to/config
+    mkdir /path/to/cache
     ```
 
-2. Start the server in one of the following ways
-
-    **Docker**
+4. Create and run a container in one of the following ways:
 
-    Running a Container is very easy (as you might have noticed while running the Hello world! Container). However when running a service you'll need to add some options to your docker run Command. Below is the full command and then it will be explained what the different parts mean.
+    **Using Docker command line interface:**
 
     ```sh
     docker run -d \
-     --volume /path/to/config:/config \
-     --volume /path/to/cache:/cache \
+     --volume jellyfin-config:/config \
```

Comment:
I personally hate volumes and recommend we advise to use mounts since someone new would probably have an easier time with backing up and maintaining mounts

Reply:
Ok lets do only bind mount.s